### ISSUE TITLE
Still getting "It appears that thegamesdb.net isn't up"

### ISSUE THREAD

**Issue by Cereal-Killa** [sselph/scraper#237]
Does it still work after the commit that changed the url to "http://legacy.thegamesdb.net"?
Downloaded the latest release but it still fails to scrape.

**Comment by darkguy2008** [comment]
Same here using Windows x64 build, all ROMs say hash not found and thinks TGDB is down when it's evidently up... any ideas?

**Comment by YouEbr** [comment]
> Same here using Windows x64 build v1.4.6, all ROMs say hash not found and thinks TGDB is down when it's evidently up... any ideas?

I am in the same boat. Any solution?

**Comment by darkguy2008** [comment]
> > Same here using Windows x64 build v1.4.6, all ROMs say hash not found and thinks TGDB is down when it's evidently up... any ideas?
> 
> I am in the same boat. Any solution?

I solved it downloading http://jujuvincebros.fr/hard-soft/tutoriels/628-arrm-another-recalbox-roms-manager ... it's a bit of a pain to make it work, but it works, which is important. General workflow is pick the correct folders, pick the platform at the top-left side, then highlight the games you want to scrape, at the bottom click TGDB's button (it has the logo) and click Save gamelist.xml at the right. The UI is horrible, but it does the job.

**Comment by J-Swift** [comment]
1.4.6 is using the old `thegamesdb.net` API. You will need to build / run using [the most recent commit](509443bf66d9fccb1d6aa2417902124bd48f2141) which should work against the new API.

**Comment by darkguy2008** [comment]
> 1.4.6 is using the old `thegamesdb.net` API. You will need to build / run using [the most recent commit](https://github.com/sselph/scraper/tree/509443bf66d9fccb1d6aa2417902124bd48f2141) which should work against the new API.

Yeah I'd bet, but there's no Windows build for that one, so Windows users are a bit on a loss here :/

**Comment by YouEbr** [comment]
Get the latest build of scrapper here
https://github.com/YouEbr/scraper4w/releases/tag/061820
